modules/imap_folders/modules.php

```diff
@@ -592,7 +592,7 @@ protected function output() {
                         <a href="#" class="select_draft_folder">'.$this->trans('Select Folder').'</a>:
                         <span class="selected_draft"></span>
                     </div>
-                    <ul class="folders draft_folder_select">
+                    <ul class="folders draft_folder_select" style="z-index : 10">
```

Comment:
It could be better if you place the style in `modules/imap_folders/site.css` for: draft_folder_select, trash_folder_select, sent_folder_select, etc., so you will not repeat it in the HTML tag.

Reply:
Okay well. I shall do so. Thanks @Shadow243